usr.bin/man/man.1

```diff
@@ -347,68 +348,65 @@ If set, enables color support.
 Program used to display files.
 .Pp
 If unset, and color support is enabled,
-.Dq Li "less -sR"
+.Ql less -sR
 is used.
 .Pp
 If unset, and color support is disabled, then
 .Ev PAGER
 is used.
 If that has no value either,
-.Dq Li "less -s"
+.Ql less -s
 is used.
 .El
 .Sh FILES
-.Bl -tag -width indent -compact
+.Bl -tag -width "/usr/local/etc/man.d/*.conf" -compact
 .It Pa /etc/man.conf
-System configuration file.
+System configuration file
 .It Pa /usr/local/etc/man.d/*.conf
-Local configuration files.
+Local configuration files
 .El
 .Sh EXIT STATUS
 .Ex -std
 .Sh EXAMPLES
 Show the manual page for
 .Xr stat 2 :
-.Bd -literal -offset indent
-$ man 2 stat
-.Ed
+.Pp
+.Dl man 2 stat
```

Comment:
Historically, `$` or `%` was placed before a command example to indicate that it does not require super user privileges.  And `#` to indicate that you must be root to execute.
It also helped with visualizing if it is a command or the output of a command.

I am not aware of this convention having fallen into disuse (maybe for copy and paste?).

Evidently, my recommendation would be to keep the prompts.

Reply:
I do not feel strongly about this. I have been asked not to add them to other pages because that breaks triple click copy paste of examples. Almost no pages have prompts.

Reply:
Barring very few exceptions, everything in section one should be a utility executable with normal privileges, and everything in section eight should be executed for administration, and elevated privileges should only be used when required for administration.

Reply:
More pages omit the prompt than include it, although there are a non-negligible number of ones that have it. Notably, a number of mandoc pages do, including mandoc's own version of man.1.

Section 1 pages that contain `^.Dl \$`:

bin/cp/cp.1
bin/ls/ls.1
bin/mkdir/mkdir.1
bin/mv/mv.1
bin/ps/ps.1
bin/rm/rm.1
bin/rmdir/rmdir.1
contrib/libfido2/man/fido2-assert.1
contrib/libfido2/man/fido2-cred.1
contrib/mandoc/apropos.1
contrib/mandoc/demandoc.1
contrib/mandoc/man.1
contrib/mandoc/mandoc.1
contrib/netcat/nc.1
crypto/openssh/ssh-keyscan.1
usr.bin/grep/grep.1
usr.bin/head/head.1
usr.bin/ktrace/ktrace.1
usr.bin/lock/lock.1
usr.bin/lockf/lockf.1
usr.bin/ssh-copy-id/ssh-copy-id.1
usr.bin/tail/tail.1
usr.bin/truss/truss.1

Reply:
That is quite noticable. I am very happy to have reverted proposed removal of prompts.